src/bin/cli/kedqr/decode/hash.rs

```diff
@@ -3,17 +3,41 @@ use catalyst_toolbox::kedqr::decode;
 use catalyst_toolbox::kedqr::QrPin;
 use chain_crypto::AsymmetricKey;
 use chain_crypto::Ed25519Extended;
+use chain_crypto::SecretKey;
 use std::fs::File;
 use std::io::BufRead;
 use std::io::Write;
+use std::path::Path;
 use std::{error::Error, fs::OpenOptions, io::BufReader, path::PathBuf};
 
 pub fn decode_hash(
     input: PathBuf,
     output: Option<PathBuf>,
     pin: QrPin,
 ) -> Result<(), Box<dyn Error>> {
-    // open input key and parse it
+    // generate qrcode with key and parsed pin
+    let secret = secret_from_hash(input, pin)?;
```

Comment:
so, this seems not to be something particular to this PR since the `decode_hash` function is already there, but the name of these things is weird, why `hash`? doesn't this decrypt the QR code payload? that is by no means a hash.

Reply:
Hm, maybe hash is not very fortunate name, `decode_method` basically returns secret key based on qr payload file. Not actual image but string representation. From business perspective we differentiate payload from qr code, as user can copy payload(i call it hash) from yoroi instead of qr image from other wallets. For other cli i had to detach logic from decode_hash, since decode_hash returns file rather than secret key structure. Is renaming sufficient fix?

Reply:
I mean, if it's terminology from yoroi or some business term maybe we should leave it. But it's certainly confusing to use _hash_ for a _ciphertext_. I mean, a hash is usually something that you don't even want to be able to decode in the first place.

Reply:
so maybe ciphertext or payload (because it's shorter) is a better name? I will rename all hash -> ciphertext or payload 

```
catalyst-toolbox kedqr decode payload 
```

or

```
catalyst-toolbox kedqr decode ciphertext 
```